### ISSUE TITLE
Move pricing to pay per use and remove auto charging

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#21]
This is more ethical.

**Comment by ngehlert** [comment]
I don't think you actually need to remove auto charging, but I'd think it would be a great option especially for small packages/pages. 
The smallest package you offer starts at <100k views. And I think small or personal pages sometimes just have couple of 100s oder 1.000s page views - in that scenario paying for 100.000 views just feels bad

**Comment by adriaanvanrossum** [comment]
Love that you're commenting here. The issue is not very clear, sorry about that. With auto charging we mean that we don't want to charge a user right after the trial period. We want to give them an option to choose when the trial ends without charging directly.

**Comment by ngehlert** [comment]
ah allright. so my comment is regarding the first part (pricing to pay per use) then :D 

haven't thought about the auto charging but it is actually a great idea 👍

**Comment by adriaanvanrossum** [comment]
This is also cool, from Netflix:

<kbd><img width="320px" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/68670941-b80c7b80-0580-11ea-9abb-583c59563df4.png"></kbd>

**Comment by adriaanvanrossum** [comment]
Design pricing to be more like this:
- https://mention.com/en/pricing/
- https://ghost.org/pricing/

<kbd><img  alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/68733623-d1a5d580-0609-11ea-86ad-f6c448f125f0.png"></kbd>

**Comment by adriaanvanrossum** [comment]
Fix not centered signup boxes on mobile (via @lastnamefirst / twitter.com/lastnamefirst) while we are at it.

<kbd><img width="320" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/68734172-f9963880-060b-11ea-95e7-010bb99d409d.png"></kbd>

**Comment by adriaanvanrossum** [comment]
Check https://baremetrics.com/blog/saas-pricing-models

**Comment by ngehlert** [comment]
wow thats a pretty interesting article. there is actually a lot of thought in pricing - or at least it can be.

so what are your take aways from the article? still want to pursue pay-per-use? or rather different tiers? or.... ? :D

**Comment by adriaanvanrossum** [comment]
I would love to combine usage-based pricing with feature-based pricing. For example fine grained user access control is something bigger companies care about and smaller users don't.

I want to prevent pages looking like this (way to many options):

<kbd><img width="1440" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/69005941-4e62e780-095b-11ea-8249-46d4697dd6f8.png"></kbd>

I just want a basic selector with some check boxes for extra features.

| page views + events |   price | with badge |
|--------------------:|--------:|-----------:|
|             100,000 |     $20 |        $10 |
|             250,000 |     $40 |        $20 |
|             500,000 |     $60 |        $30 |
|             750,000 |     $80 |        $40 |
|           1,000,000 |    $100 |        $50 |
|         >10,000,000 | `M` * $50 |    `M` * $25 |
|        >100,000,000 | `M` * $40 |    `M` * $20 |
|      >1,000,000,000 | `M` * $30 |    `M` * $15 |

> `M` = 1 Million

Extra features:
- High grained user admin (unlimited users) $20/m
- Custom domain (bypass ad-blockers) $10/m

Discounts:
- Yearly gives you 2 months for free
- Placing a badge on your website gives you 50% discount (except for the extra features).

I did a bit or calculations on my current customers:

<kbd><img width="1440" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/69036002-7dd72a00-0a17-11ea-8969-621b4dd3fe84.png"></kbd>

Note that it only shows customers with 10,000 page views or more because I will charge below 10,000 the same as 10,000.

> If you want another article, love this one as well: https://www.joelonsoftware.com/2004/12/15/camels-and-rubber-duckies/ (thanks to @tehnlulz)

**Comment by adriaanvanrossum** [comment]
Make sure to include issue https://github.com/simpleanalytics/roadmap/issues/385 here as well.

**Comment by adriaanvanrossum** [comment]
Feedback from a beloved ex-customer:

> Thanks. If you are working on a change, I'll give a bit more flavor into why I left. It may sound like a complaint, but I think it's fixable.
> A few months ago I recieved an e-mail from you (maybe automated) that I had been automatically bumped up to the next tier and that my credit card would be charged at 59 rather than 19. I'm sure it's within the agreement we entered into, but I do not like the feeling that someone else has a carte blanche to charge my credit card what they want.
> I saw that the next tier was "enterprise", with an unknown price. I knew my credit card would expire in a few months, before I could reach the enterprise level. If I would have been in doubt, I would probably have cancelled my subscription immediately. Now I rather waited for it to happen by itself.
>
> Two things would have helped a lot:
> - More than 1 weeks notice
> - A link I could press in the e-mail to accept the new tier (so I feel in control)
> 
> I might of course be the odd one, and maybe more of your customers convert by the automated upgrade rather than an opt-in upgrade. In which case you should obviously keep it :) All the best from ...
